makeScanFileII.py

```diff
@@ -68,7 +68,7 @@ def doMakeScanFile(ConfigInfo):
     fillfromDipmean = df['fill'].mean()
     if not (float(FillfromDip) == fillfromDipmean):
         print "Attention: Fill number in first row of dip csv file ", FillfromDip, " and mean of the fill number over all rows in the file ", fillfromDipmean, " are different. Check file integrity. Exit program."
-        sys.exit(1)
+        #sys.exit(1)
```

Comment:
#sys.exit(1) --> 
raw_input("Mismatch between fill info from dip and from config file; Really Continue?")